docs/en/guides/How-to-release.md

```diff
@@ -79,18 +45,18 @@ git push origin v${RELEASE_VERSION}
 ## Stage the release 
 
 ```bash
-./mvnw flatten:flatten install deploy -DskipTests
+./mvnw package -DskipTests
 ```
 
 This command will build, sign, and deploy the release to the Apache staging repository.
 You should be prompted for your GPG passphrase during the deployment process.
 If no prompt is present, you can set the passphrase in the environment variable `MAVEN_GPG_PASSPHRASE`:
 
 ```bash
-MAVEN_GPG_PASSPHRASE=<your-gpg-passphrase> ./mvnw flatten:flatten install deploy -DskipTests
```

Comment:
GPG related is not needed anymore, it was for maven verification. Now we will only sign the final tar.